CONTRIBUTING.md

```diff
@@ -122,32 +122,49 @@ command inside repository root directory:
 docker build -f Dockerfile -t yacgroup/yacguide-build .
 ```
 
-## Building the APK locally
+## Building the APK
```

Comment:
Probably not really because most of the people would just use Android Studio. I will rework the documentation anyway, so I will think about this again.